src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -1233,23 +1233,6 @@ private uint EventUnregister(long registrationHandle)
         {
             return m_eventProvider.EventUnregister(registrationHandle);
         }
-
-        static int[] nibblebits = { 0, 1, 1, 2, 1, 2, 2, 3, 1, 2, 2, 3, 2, 3, 3, 4 };
-        private static int bitcount(uint n)
-        {
-            int count = 0;
-            for (; n != 0; n = n >> 4)
-                count += nibblebits[n & 0x0f];
-            return count;
-        }
-        private static int bitindex(uint n)
-        {
-            Debug.Assert(bitcount(n) == 1);
```

Comment:
Note that this assert is already taken care of in the preceding `if` statement at the callsite